ci-operator/jobs/openshift/release/openshift-release-periodics.yaml

```diff
@@ -600,6 +600,8 @@ periodics:
     org: openshift
     repo: release
   decorate: true
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
```

Comment:
I don't think we want to rehearse the bootstrapper and get our cluster config changed from random PRs?

Reply:
addressed